### PR TITLE
init mysql database

### PR DESCRIPTION
**Pull request by lht6**
It is to init mysql for EF core.

### CODE REVIEW

File: Controllers/ValuesController.cs [hunk 1]
```diff
@@ -28,6 +30,17 @@ public ActionResult<string> Get(int id)
         [HttpPost]
         public ActionResult<string> Post([FromBody] string value)
         {
+            using (var context = new ApplicationDbContext())
+            {
+                var user = new User()
+                {
+                    Name = "Bill"
```
Comment (ZhengAudrey) [inline Controllers/ValuesController.cs]: can you confirm?

File: appsettings.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "server=localhost;port=3306;database=jbpdb;uid=root;password=123456"
```
Comment (ZhengAudrey) [inline appsettings.json]: should use ip address

File: Controllers/ValuesController.cs (hunk 1 above)
Comment (lht6) [reply Controllers/ValuesController.cs]: confirmed with PM.

File: appsettings.json (hunk 2 above)
Comment (lht6) [reply appsettings.json]: do this next time.

**Comment by lht6** [comment]
it is for Mysql EF

**Pull request merged**